toolflow/vivado/platform/pcie/pcie_base.tcl

```diff
@@ -281,28 +281,10 @@
     }
 
     set out_ic [tapasco::ip::create_axi_sc "out_ic" 1 4]
-    tapasco::ip::connect_sc_default_clocks $out_ic "design"
-
-    if {$device_type != "virtexuplus" && $device_type != "virtexuplusHBM"} {
-      if { $pcie_width == "x8" } {
-        puts "Using PCIe IP for x8..."
-        set bridge [tapasco::ip::create_pciebridgetolite "PCIeBridgeToLite"]
-      } else {
-        puts "Using PCIe IP for x16..."
-        set bridge [tapasco::ip::create_pciebridgetolite_x16 "PCIeBridgeToLite"]
-      }
-        connect_bd_intf_net [get_bd_intf_pins -regexp $pcie/M_AXI(_B)?] \
-      [get_bd_intf_pins -of_objects $bridge -filter "VLNV == [tapasco::ip::get_vlnv aximm_intf] && MODE == Slave"]
-
-      connect_bd_net [get_bd_pins axi_pcie3_0/axi_aclk] [get_bd_pins -of_objects $bridge -filter "TYPE == clk"]
-      connect_bd_net [get_bd_pins axi_pcie3_0/axi_aresetn] [get_bd_pins -of_objects $bridge -filter "TYPE == rst"]
+    tapasco::ip::connect_sc_default_clocks $out_ic "host"
```

Comment:
Why are the BridgeToLite-IPs no longer required? How is this related to the new runtime?

Reply:
Because the Rust functions I used are much more efficient and use e.g. SSE transfers whenever available they will break the forced conversion to 32 Bit/64Bit which we used previously.